src/lbaf/IO/configValidator.py

```diff
@@ -0,0 +1,123 @@
+from logging import Logger
+
+import yaml
+from schema import And, Optional, Or, Schema, Use
+
+from lbaf.Utils.logger import logger
+
+
+class ConfigValidator:
+    """ Validates schema of VT Object Map files (json)
+    """
+    def __init__(self, config_to_validate: dict, logger: Logger):
+        self.__config_to_validate = config_to_validate
+        self.__skeleton = Schema({
+            Or("from_data", "from_samplers", only_one=True): dict,
+            "work_model": {"name": And(str, lambda c: c in ("LoadOnly", "AffineCombination"),
+                                       error="LoadOnly or AffineCombination needs to be chosen!"),
+                           "parameters": {"alpha": float,
+                                          "beta": float,
+                                          "gamma": float}},
+            "algorithm": {
+                "name": And(str, lambda d: d in ("InformAndTransfer", "BruteForce"),
+                            error="InformAndTransfer or BruteForce needs to be chosen!"),
+                Optional("parameters"): dict
+            },
+            "exodus": bool,
+            "x_procs": int,
```

Comment:
These variables should **not** be defined by default: `x_procs`, `y_procs`, `z_procs`. They are not used when `exodus` is OFF.